tests/platform_tests/api/conftest.py

```diff
@@ -29,11 +31,13 @@ def start_platform_api_service(duthost, localhost):
         duthost.copy(src=src_path, dest=dest_path)
         duthost.command('docker cp {} pmon:{}'.format(dest_path, pmon_path))
 
+        duthost.command('systemctl stop caclmgrd.service')
+        duthost.command('iptables -F')
```

Comment:
is this flush? I think this might cause some issue since there are other iptable rules.

Reply:
Yes it is a flush. If no table name is specified (like here), it will flush all chains in the "filter" table; it will not affect other tables. caclmgrd also performs a flush whenever there is a rule change, so this would also be a problem in production. Are you referring to the "mangle" table? This is not affected, as this only flushes the "filter" table.

Reply:
Nevertheless, I do believe that flushing all rules is a bit of overkill. So I have refactored the code to instead prepend an iptables rule to allow traffic destined for the HTTP server's port.